scss/mixins/_border-radius.scss

```diff
@@ -33,3 +33,11 @@
     border-bottom-left-radius: $radius;
   }
 }
+
+// Rounded pill used for utility class
+
+@mixin rounded-pill($radius) {
+  @if $enable-rounded {
```

Comment:
I don't think the mixin is necessary, if someone adds the utility or modifying class, I'm pretty sure they want the rounded effect. `.rounded-circle` doesn't have this either.

Reply:
We could simply have: `.btn-rounded` without a mixin? Especially if removing the one from the docs page for utilities.